src/components/app-layout.tsx

```diff
@@ -17,9 +20,7 @@ export function AppLayout({ children, links }: { children: ReactNode; links: { l
         <main className="flex-grow container mx-auto p-4">
           <ChainChecker>
             <div />
-            {/*  <AccountChecker />*/}
```

Comment:
Please don't remove unrelated code, this is an actual placeholder I'll add back in.